pkg/sqlite/query.go

```diff
@@ -113,6 +113,76 @@ func (s *SQLQuerier) GetPackage(ctx context.Context, name string) (*registry.Pac
 	return pkg, nil
 }
 
+func (s *SQLQuerier) GetDefaultPackage(ctx context.Context, name string) (string, error) {
```

Comment:
nit: should this be `GetDefaultChannel`?

Reply:
Also - is the query interface updated with these?

Reply:
latest updated includes the interface update